orderer/consensus/etcdraft/blockpuller.go

```diff
@@ -175,7 +175,8 @@ func NewBlockFetcher(support consensus.ConsenterSupport,
 			}
 		},
 		BlockSourceFactory: func(c cluster.FetcherConfig) cluster.BlockSource {
-			return &cluster.BlockPuller{VerifyBlockSequence: verifyBlockSequence,
```

Comment:
Although not checked in this fork's CI, this fails the basic linter checks in the main repo.